### DISCUSSION TITLE
Configuration TAB not open after build!!! only the configuration tab!!!???

### DISCUSSION THREAD

**Discussion by jmcdrone** [iNavFlight/inav-configurator#2213] [General]
I build myself a .hex file from omnibusF4V3  on visual code without any error. But on the inav configurator i could not open the configuration tab. All other tabs are normally working!! Is that a bug? Best Regards

**Comment by sensei-hacker** [comment]
Which version number did you build, and what did you change?

**Comment by jmcdrone** [comment]
I use the code from github and run the command make OMNIBUSF4V3 on visual
code. I get a .hex file and flash to my omnibusf4 flight controller. I
change the softserial from s6 to a led pad. It works but i have to use cli
to activate softserial feature because the configuration tab doesn't open
on inav configurator

Em qui., 17 de out. de 2024 13:45, Sensei ***@***.***>
escreveu:

**Comment by jmcdrone** [comment]
I build the newest version

Em qui., 17 de out. de 2024 14:16, juliano cortez ***@***.***>
escreveu:

**Comment by MrD-RC** [comment]
Are you using the configurator from master? It will not work with the 7.1.2 release configurator.

**Reply by jmcdrone** [reply]
Yes, I use master. Which one may I use then?

**Reply by sensei-hacker** [reply]
> Yes, I use master. Which one may I use then?

Then you would need the master of Configurator, which may have bugs, because it's not release code.

You might be better off making your change to 7.1.2

Also if you haven't seen this document, we have alrerady provided targets with soft serial on three different choices of pins:
https://github.com/iNavFlight/inav/blob/master/docs/boards/Omnibus%20F4.md

**Reply by jmcdrone** [reply]
where could i found the master configurator or the 1.7.2 code ? the point is that i am using one servo there for i need change the code from myself.

**Reply by sensei-hacker** [reply]
> where could i found the master configurator or the 1.7.2 code ? the point is that i am using one servo there for i need change the code from myself.


On GitHub, on the top left there's a drop down to choose between different branches and tags. Click on the tags section.  Or if you clicked the git repo, do "git checkout 7.1.2"

**Reply by jmcdrone** [reply]
ok obrigado!
